Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>
             <li>Reviews</li>
             <li>Projects</li>

```

Comment:
Consider wrapping the image and text within a container element like a div for better structure and styling.